HIP/hip-260.md

```diff
@@ -122,6 +126,63 @@ final updated value of the storage slot after the completion of the smart
 contract call. Transient states between the start and finish of the contract are
 not stored in the record stream.
 
+### Record Stream File Changes
+
+Currently Hedera nodes produce record stream files that contain a series of
+transactions within a two second interval. The record stream file (.rcd) 
```

Comment:
Probably we can describe that "an interval with given amount of X seconds" is used to determine the contents of a given record file/block. In this way we would be more compliant if in the future we change the two second interval with four second interval for example.

Reply:
Changed to "preconfigured interval"